docs/source/services/timelock_service/migration.rst

```diff
@@ -16,137 +11,21 @@ this guarantee when moving from an embedded timestamp service to an external tim
-      As fast-forward is idempotent (it sets the timestamp to be the maximum of its current value and the
-      ``newMinimum``), *if clients have not been started again* this can be fixed by doing another fast-forward.
-
-To verify that this step was completed correctly, you may curl the Timelock Server's fresh-timestamp endpoint.
-
-   .. code:: bash
-
-      curl -XPOST localhost:8080/test/timestamp/fresh-timestamp
-
-The value returned should be (assuming no one else is using the Timelock Server) 1 higher than ``TS``.
-
-Step 4: Invalidating the Old AtlasDB Timestamp
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The steps for invalidating the old AtlasDB timestamp will vary, depending on your choice of underlying key value store.
-
-- If using Postgres or Oracle, it suffices to rename the relevant column in the timestamp table (use ``ALTER TABLE``).
-  For example, for Postgres:
-
-     .. code:: sql
-
-        ALTER TABLE atlasdb_timestamp RENAME last_allocated TO LEGACY_last_allocated;
-
-- If using Cassandra, one method of invalidating the table is to overwrite the timestamp bound record with the
-  empty byte array (consider using ``cqlsh`` to do this). This table is stored in the same keyspace that your
-  AtlasDB client uses for its key-value service.
-
-     .. code:: bash
-
-        SELECT * FROM atlasdb."_timestamp";
-        <note the value returned by this - call this K>
-        INSERT INTO atlasdb."_timestamp" (key, column1, column2, value) VALUES (0x7472, 0x7472, -1, K);
-        INSERT INTO atlasdb."_timestamp" (key, column1, column2, value) VALUES (0x7473, 0x7473, -1, 0x);
-
-- Dropping the table, generally speaking, will *not* work (on the next startup of an embedded Timestamp Service,
-  AtlasDB will believe it is starting up the Timestamp Service for the first time, and thus start again from 1).
-- Setting the value to ``Long.MAX_VALUE`` or ``Long.MIN_VALUE`` will not work (Java Longs do not throw on arithmetic
-  overflow, and although ordinarily the first timestamp AtlasDB issues is 1 we do not throw on negative numbers).
-
-Please contact the AtlasDB team for assistance if you are uncertain about this step or otherwise run into difficulties.
-
-To verify that this step was completed successfully, you may restart one of your AtlasDB clients. This should fail when
-TransactionManagers.create() is called, throwing a runtime exception.
-
-Steps 5 and 6: Client Configuration and Restart
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Configure your clients to use the Timelock Server following the instructions in :ref:`timelock-client-configuration`.
-You may then restart your clients; they should now communicate with the Timelock Server when requesting timestamps
-and locks. This completes the migration process.
-
 Automated Migration
 -------------------
 
-The AtlasDB team is currently working on an automated migration process, such that the steps above are run when one
```

Comment:
I think we should insert a warning or even danger block here that Automated Migrations only works for Cassandra at the moment; if you're using DBKVS or JDBC or something else you need to do the migration on your own.